core/src/test/java/org/apache/druid/timeline/partition/DimensionRangeShardSpecTest.java

```diff
@@ -136,6 +136,61 @@ public void testShardSpecLookup()
     );
   }
 
+  @Test
+  public void testShardSpecLookupWithNull()
+  {
+    setDimensions("dim1", "dim2");
+
+    final DimensionRangeShardSpec shard0 = new DimensionRangeShardSpec(
+        dimensions,
+        null,
+        StringTuple.create("India", null),
+        1,
+        1
+    );
+
+    final DimensionRangeShardSpec shard1 = new DimensionRangeShardSpec(
+        dimensions,
+        StringTuple.create("India", null),
+        StringTuple.create("Spain", "Valencia"),
+        10,
+        1
+    );
+
+    final DimensionRangeShardSpec shard2 = new DimensionRangeShardSpec(
+        dimensions,
+        StringTuple.create("Spain", "Valencia"),
+        StringTuple.create("Tokyo", null),
+        10,
+        1
+    );
+
+    final DimensionRangeShardSpec shard3 = new DimensionRangeShardSpec(
+        dimensions,
+        StringTuple.create("Tokyo", null),
+        null,
+        100,
+        1
+    );
+    final ShardSpecLookup lookup = shard0.getLookup(Arrays.asList(shard0, shard1, shard2, shard3));
```

Comment:
Is this call still valid?

Reply:
I learned this from somewhere else, like HashBucketShardSpecTest.testShardSpecLookup

Reply:
Any problem with this line?

Reply:
Ah, yes, you are right. I was under the impression that the `getLookup` method was not defined on `DimensionRangeShardSpec` but it is there in the super class.